docs/wiki/Manual-Testing.md

```diff
@@ -142,7 +143,7 @@ When doing a test pass, copy this page and consider using these status indicator
 | | :mag: **Navigation Bars**<br />- Open some existing source files, make sure you can navigate around the file choosing classes or methods.<br />- Switch between project contexts<br />- In VB, the NavBar can do code generation for events and for New/Finalize methods | | | |
 | | **Metadata As Source**<br />- Press F12 on a predefined type and expect the cursor to move the predefined type definition inside a Metadata-As-Source Generated document.<br />- Expect to see the xml doc comments collapsed above the method. | | | N/A |
 | | **Navigate to Decompiled Source** [Visual Studio 2017 version 15.6 Preview 2]<br />- Set the option at `Tools -> Options -> Text Editor -> C# -> Advanced -> Enable navigation to decompiled sources (experimental)` <br />- F12 on a type defined in metadata<br />- You should see decompiled method bodies, not just declarations | | | N/A |
-| | :mag: **Navigate To**<br />- Place caret on a user defined Type reference and press "ctrl + ,"<br />- This should list the User Defined Type in the drop down on the Upper Right corner of the editor and selecting this item will move the cursor to the User Reference Definition<br />- Filters per kind of symbol [Visual Studio 2017]<br />- Results should be sorted as follows: [Visual Studio 2017 version 15.6 Preview 4]<br /><details><summary>Expand for details</summary>TODO</details> | | | |
+| | :mag: **Navigate To**<br />- Place caret on a user defined Type reference and press "ctrl + ,"<br />- This should list the User Defined Type in the drop down on the Upper Right corner of the editor and selecting this item will move the cursor to the User Reference Definition<br />- Filters per kind of symbol [Visual Studio 2017]<br />- Results should be sorted as follows: [Visual Studio 2017 version 15.6 Preview 4]<br /><details><summary>Expand for details</summary>TODO</details><br />- Records support in navigate to | | | |
```

Comment:
Thanks. I'd missed that we already had a section for Navigate To.

Reply:
@jcouv Given that the linked PR is merged, do we still need this?

Reply:
Nah, all good. I didn't realize we had this line already. Thanks

Reply:
> 
> 
> Nah, all good. I didn't realize we had this line already. Thanks

Ok I'll close.